versoview_messages/src/lib.rs

```diff
@@ -145,6 +151,9 @@ impl Default for ConfigFromController {
             maximized: false,
             visible: true,
             focused: true,
+            decorated: false,
+            transparent: true,
```

Comment:
Curious why `transparent` is default to true

Reply:
This is to align with our old behavior

> ```rust
>         let window_attrs = WinitWindow::default_attributes()
>             .with_decorations(false)
>             .with_transparent(true);
> ```